projects/front-end-library/src/lib/components/modal/angular/modal.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+    selector: 'bf-modal',
+    templateUrl: './modal.component.html',
+})
+
+export class ModalComponent implements OnInit {
+    constructor() {
+    }
+
+    @Input() id: string;
+    @Input() modalSize: 'sm' | 'md' | 'lg';
+    @Input() modalAnimation: string;
+    @Input() modalLabelledBy: string;
+    @Input() modalTheme: string;
+
+    @Input() bgImage: string;
+    @Input() bgLgImage: string;
+    @Input() bgMdImage: string;
+    @Input() modal_header: string | 'TwigBlock';
+    @Input() modalTitle: string;
+    @Input() modalSubtitle: string;
+
+    /** Content section takes as much height as it needs */
+    @Input() modal_body_subcontainer: string | 'TwigBlock';
+    /** Content section has a maximum height so it becomes scrollable */
+    @Input() modal_body_scroll_container: string | 'TwigBlock';
+    @Input() modalBudy: string;
+
+    @Input() modal_footer: string | 'TwigBlock';
+
+    /** Twig only */
+    @Input() class: string;
+    @Input() modalContentCLass: string;
```

Comment:
```suggestion
    @Input() modalContentClass: string;
```

A moins que ce soit comme ça dans le UI-Kit?

Reply:
c'est du copié-collé de UIKit. perso ça serait contentClass tout court. à la bifrostisation on fait un gros ménage de namming. garde tes idées pas trop loin